sys-apps/asahi-scripts/asahi-scripts-20240822.ebuild

```diff
@@ -6,19 +6,34 @@ EAPI="8"
 DESCRIPTION="Apple Silicon support scripts"
 HOMEPAGE="https://asahilinux.org/"
 SRC_URI="https://github.com/AsahiLinux/${PN}/archive/refs/tags/${PV}.tar.gz -> ${PN}-${PV}.tar.gz"
-
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="~arm64"
+KEYWORDS="arm64"
 
-PATCHES=(
-	"${FILESDIR}/makefile.patch"
-	"${FILESDIR}/update-m1n1-dtbs.patch"
-)
+BDEPEND="
+	dev-build/make
+	virtual/udev
```

Comment:
What's the BDEPEND on udev for here? You can also drop dev-build/make as it's in `@system`.

Reply:
The BDEPEND on udev is for the battery charge control behaviour rules. I assume this should just go in RDEPEND instead though? We just don't want to take ownership of `/etc/udev/` or `/usr/lib/udev/`

Reply:
Yeah, it belongs in RDEPEND then.